2 table.sql

```diff
@@ -0,0 +1,30 @@
+use discoveryou
+go 
+drop table if exists dbo.Personalities
+go 
+create table dbo.Personalities(
+    PersonalityId int not null IDENTITY primary key,
+    ClientName varchar(50) not null unique,
+    DOB date not null ,
+    DateTestTaken date not null constraint c_personalities_datetakentest_must_be_before_Jan_24_2021 check(DateTestTaken > '1-24-2021'),
```

Comment:
change the message name to datetesttaken must be after Jan 24 to prevent confusion.